python/lsst/dax/webservcommon/auth.py

```diff
@@ -0,0 +1,72 @@
+import json
```

Comment:
Module-level docstring and copyright are sorely missed :cat2:

Reply:
added